public/css/filter.css

```
@@ -0,0 +1,100 @@
+.filter {
+    display: flex;
+    flex-direction: row;
+    justify-content:flex-start;
+    align-items:center;
+}
+
+.search-bar {
+    margin: 10px;
+}
+
+.dropdown {
+    background-color: #eee;
+    display: flex;
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+  }
```

```suggestion
}
```
Every closing bracket are wrong indented